seqr/views/apis/variant_search_api.py

```diff
@@ -309,22 +310,24 @@ def search_context_handler(request):
         projects = Project.objects.filter(projectcategory__guid=context.get('projectCategoryGuid'))
     elif context.get('searchHash'):
         try:
-            results_model = _get_or_create_results_model(context['searchHash'], context.get('searchParams'), request.user)
+            results_model = _get_or_create_results_model(context['searchHash'], context.get('searchParams'),
+                                                         request.user, session=request.session['anvil'])
         except Exception as e:
             return create_json_response({'error': str(e)}, status=400, reason=str(e))
         projects = Project.objects.filter(family__in=results_model.families.all()).distinct()
     else:
         error = 'Invalid context params: {}'.format(json.dumps(context))
         return create_json_response({'error': error}, status=400, reason=error)
 
-    response.update(_get_projects_details(projects, request.user, project_category_guid=context.get('projectCategoryGuid')))
+    response.update(_get_projects_details(projects, request.user, session=request.session['anvil'],
+                                          project_category_guid=context.get('projectCategoryGuid')))
 
     return create_json_response(response)
 
 
-def _get_projects_details(projects, user, project_category_guid=None):
+def _get_projects_details(projects, user, session=None, project_category_guid=None):
     for project in projects:
-        check_project_permissions(project, user)
+        check_project_permissions(project, user, session)
```

Comment:
Codacy found an issue: [Too many positional arguments for function call](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=6211929)